src/Files.App/MainWindow.xaml.cs

```diff
@@ -27,6 +28,9 @@ public sealed partial class MainWindow : WindowEx
 
 		public IntPtr WindowHandle { get; }
 
+		[DllImport("user32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+		static extern bool SetPropW(IntPtr hWnd, string lpString, IntPtr hData);
```

Comment:
Could you use Vanara's User32 class instead?

Reply:
Is there a benefit to that? I though adding more references to Vanara would make it harder to enable trimming.

Reply:
You've got a point. Could you move it then to `InteropHelpers`?

Reply:
Good suggestion, I addressed it in the last commit.